app/controllers/application_controller/ci_processing.rb

```diff
@@ -440,7 +464,7 @@ def deletevms
   # Import info for all selected or single displayed vm(s)
   def syncvms
     assert_privileges(params[:pressed])
-    vm_button_operation('sync', 'for Virtual Black Box synchronization')
+    vm_button_operation('sync', _('for Virtual Black Box synchronization'))
```

Comment:
`for Virtual Black Box synchronization does not apply to this item`
`No items were selected for for Virtual Black Box synchronization`

.. we probably need a better string here..